eli5/formatters/html.py

```diff
@@ -27,13 +28,15 @@
     remaining_weight_color=lambda ws, w_range, pos_neg:
         format_hsl(remaining_weight_color_hsl(ws, w_range, pos_neg)),
     format_feature=lambda f, w, hl: _format_feature(f, w, hl_spaces=hl),
+    format_value=format_value,
     format_decision_tree=lambda tree: _format_decision_tree(tree),
 ))
 
 
 def format_as_html(explanation, include_styles=True, force_weights=True,
                    show=fields.ALL, preserve_density=None,
-                   highlight_spaces=None, horizontal_layout=True):
+                   highlight_spaces=None, horizontal_layout=True,
+                   show_feature_values=True):
```

Comment:
Yeah I like the idea of having explanations in the header, and I'd also add a little "?" to indicate that there is some help there.

I agree about the default, let's keep it False as it was before, and we can mention that option in "Contribution" help.